central/metrics/central.go

```diff
@@ -191,17 +191,121 @@ var (
 		Name:      "pipeline_panics",
 		Help:      "A counter that tracks the number of panics that have occurred in the processing pipelines",
 	}, []string{"resource"})
+
 	sensorConnectedCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: metrics.PrometheusNamespace,
 		Subsystem: metrics.CentralSubsystem.String(),
 		Name:      "sensor_connected",
 	}, []string{"ClusterID", "connection_state"})
+
+	grpcMaxMessageSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.CentralSubsystem.String(),
+		Name:      "max_message_size_sent",
+		Help:      "A gauge for maximum message size seen when sending messages to sensor",
+	}, []string{"Type"})
+
+	grpcSentSize = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.CentralSubsystem.String(),
+		Name:      "grpc_message_size_sent",
+		Help:      "Histogram of sent message sizes from Central",
+		Buckets: []float64{
+			1,
+			10,
+			50,
+			100,
```

Comment:
I think, messages smaller than 1B are not realistic :)
Are we really interested how many messages < 100B arrived? I think we could easily start at 500B.

(applies to all buckets)